localization/english/culture/wc_cultural_maa_innovations_l_english.yml

```
@@ -18,3 +19,187 @@
+ innovation_wyvern_desc:0 "Wyverns share a common ancestry with both dragons and gryphons and hail from some of the most inhospitable regions of Kalimdor. Our tamers have successfully learned how to communicate with and tame these flying creatures."
+
+ #####################
+ # High Medieval Era #
+ #####################
+ 
+ innovation_blackrock_legionnaire:0 "Blackrock Legionnaires"
+ innovation_blackrock_legionnaire_desc:0 "Legionnaires are extremely strong and wield their weapons very efficiently and destructively. Compared to other Orcs, they are extremely disciplined and can work cohesively as a unit."
+
+ innovation_riflemen_dwarf:0 "Dwarven Riflemen"
+ innovation_riflemen_dwarf_desc:0 "Riflemen have always been among the faithful protectors of their mountain kingdom, Khaz Modan. Carrying the legendary single-shot blunderbuss long rifles, they are excellent marksmen and can hit enemies on land and in the air. They are famous for their masterful skills and toughness."
+
+ innovation_riflemen_human:0 "Human Riflemen"
+ innovation_riflemen_human_desc:0 "Riflemen are highly skilled sharpshooters armed with rifles. While dwarves were among the first to invent and use them, many races since took up these weapons and made them their own."
+  
+ innovation_shredder:0 "Shredders"
+ innovation_shredder_desc:0 "Goblin shredders are true masterpieces of goblin engineering. They are large machines equipped with a saw, making them masterful lumberjacks. When it comes to war, these armored constructions can be used in the front line to mow down potential enemies one by one."
+ 
+ innovation_steamwarrior:0 "Steamwarriors"
+ innovation_steamwarrior_desc:0 "In battle steam warriors are fearless and resolute, capable of enduring the withering fire of musket, spell or arrow. Their skill at piloting their steam armor allows them to move gracefully and with tremendous speed, reacting faster than one would think possible."
+ 
+ innovation_technomage:0 "Technomages"
+ innovation_technomage_desc:0 "The technomage has little patience when it comes to battle. They will go in, hit the enemy with all they have, and have it done with. The technomage knows how to connect with their technological weapons, channeling their magic through them. Their gun does not shoot bullets, but fireballs or dire rats. The technomage can, by touch, place any spell they have prepared into a technological weapon."
+ 
+ #####################
+ # Late Medieval Era #
+ #####################
+ 
+ innovation_goblin_air:0 "Goblin Aerial School"
+ innovation_goblin_air_desc:0 "The ingenious goblin inventors found a way to create these giant bags filled with a gas lighter than air and attached a gondola to its belly. Just like their gnomish counterparts they could not resist putting guns and explosives to these vehicles, turning them into a hazard on every battlefield.
+ With our unique aerial armaments, we have produced regiments like the Gyrocopters and the Zeppelins."
```

I think the separated lines caused your problem @arithon .
```suggestion
 innovation_goblin_air:0 "Goblin Aerial School"
 innovation_goblin_air_desc:0 "The ingenious goblin inventors found a way to create these giant bags filled with a gas lighter than air and attached a gondola to its belly. Just like their gnomish counterparts they could not resist putting guns and explosives to these vehicles, turning them into a hazard on every battlefield.
 With our unique aerial armaments, we have produced regiments like the Gyrocopters and the Zeppelins."
```